community/modules/scheduler/schedmd-slurm-gcp-v6-controller/modules/slurm_files/main.tf

```
@@ -90,25 +88,10 @@ locals {
     # Providers
     endpoint_versions = var.endpoint_versions
 
-    # Extra-files MD5 hashes
-    # Makes config file creation depend on the files
-    # Allows for informed updates & checks on slurmsync side
-    slurm_gcp_scripts_md5 = google_storage_bucket_object.devel.md5hash,
-    controller_startup_scripts_md5 = {
-      for o in values(google_storage_bucket_object.controller_startup_scripts) : trimprefix(o.name, local.tp) => o.md5hash
-    }
-    nodeset_startup_scripts_md5 = {
-      for o in values(google_storage_bucket_object.nodeset_startup_scripts) : trimprefix(o.name, local.tp) => o.md5hash
-    }
-    login_startup_scripts_md5 = {
-      for o in values(google_storage_bucket_object.login_startup_scripts) : trimprefix(o.name, local.tp) => o.md5hash
-    }
-    prolog_scripts_md5 = {
-      for o in values(google_storage_bucket_object.prolog_scripts) : trimprefix(o.name, local.tp) => o.md5hash
-    }
-    epilog_scripts_md5 = {
-      for o in values(google_storage_bucket_object.epilog_scripts) : trimprefix(o.name, local.tp) => o.md5hash
-    }
+    # Extra-files MD5 hashes. Makes config file creation to wait on those files
```

```suggestion
    # Extra-files MD5 hashes. Makes config file creation to wait on those files
    slurm_gcp_scripts_md5 = data.archive_file.slurm_gcp_devel_zip.output_md5
```

Data resource output_md5 is known at the plan stage, so this won't make the config.yaml file known only after apply.

---

There is another motivation to include it. Our archive composing is not discriminatory so it will include garbage files (e.g. mypy cache), so it creates unnecessary diff. 
Or may have false-positivie difference in case deployments are performed from different machines.

---

As long as we are not checking for `slurm_gcp_devel.zip` in `slurmsync.py` I'm fine with removal of this